lib/gds-sso/controller_methods.rb

```diff
@@ -1,11 +1,19 @@
 module GDS
   module SSO
+    class PermissionDeniedError < StandardError
+    end
+
     module ControllerMethods
-      class PermissionDeniedException < StandardError
+      # TODO: remove this for the next major release
+      class PermissionDeniedException < GDS::SSO::PermissionDeniedError
```

Comment:
```suggestion
      class PermissionDeniedException < PermissionDeniedError
```

Just realised since the name of the exception is not a clash we don't need the `GDS::SSO` prefix

Reply:
It won't have issues since this line is under GDS::SSO::ControllerMethods and the PermissionDeniedError is under GDS:SSO? The tests still pass when i change it, but i usually play it safe when we call things under different namespaces.

Reply:
Yup, we're already in the GDS::SSO namespace so need to qualify.

I suggested the namespace as I thought we'd have both GDS::SSO::PermissionDeniedException and GDS::SSO::ControllerMethods::PermissionDeniedException